pages/index.js

```diff
@@ -1,7 +1,75 @@
+import shoppingItems from "@/assets/shopping-items.json";
+import ShoppingItemList from "@/Components/ShoppingItemList";
+import styled from "styled-components";
+
 export default function HomePage() {
   return (
-    <div>
-      <h1>Hello from Next.js</h1>
-    </div>
+    <>
+      <StyledHeader>
+        <StyledHeadline>GrocerEase</StyledHeadline>
+        <StyledContainer>
+          <StyledListName>Shopping List</StyledListName>
+          <StyledTotalItems>
+            {shoppingItems.length} items total{" "}
+          </StyledTotalItems>
+        </StyledContainer>
+      </StyledHeader>
+      <main>
+        <ShoppingItemList shoppingItems={shoppingItems} />
+      </main>
+    </>
   );
 }
+
+const StyledHeader = styled.header`
+  position: fixed;
+  top: 0;
+  width: 100%;
+  text-align: center;
+  border-radius: 0 0 15px 15px;
+  background-color: #fff4e9;
+  color: #362f23;
+  z-index: 1000;
+  margin: 0;
+`;
+
+const StyledHeadline = styled.h1`
+  top: 0;
+  width: 100%;
+  text-align: center;
+  color: #362f23;
+  border-style: solid;
+  border-bottom: 0;
+  padding: 7px 0;
+  z-index: 1000;
+  margin: 0;
```

Comment:
You might not need `top`, `width` and `z-index` here. Try it out. :) 
```suggestion
  text-align: center;
  color: #362f23;
  border-style: solid;
  border-bottom: 0;
  padding: 7px 0;
  margin: 0;
```

Reply:
Thanks! Was a careless error, thought it found every one of them.